aip_xx1_launch/launch/radar.launch.xml

```diff
@@ -10,24 +9,16 @@
     <group>
       <push-ros-namespace namespace="front_center"/>
 
-      <!-- socket can -->
-      <group>
-        <include file="$(find-pkg-share pacmod3)/launch/pacmod3.launch.xml">
-          <arg name="use_socketcan" value="true" />
-          <arg name="socketcan_device" value="$(var radar_can_device)" />
-          <arg name="namespace" value="" />>
-        </include>
-      </group>
-
       <!-- tf launch (temporary) -->
-      <node pkg="tf2_ros" exec="static_transform_publisher" name="base_link_to_ars408" output="screen" args="3.8 0.0 0.0 0.0 0.0 0.0 base_link ars408" />
+      <node pkg="tf2_ros" exec="static_transform_publisher" name="base_link_to_ars408" output="screen" args="3.8 0.0 0.0 0.0 0.0 0.0 base_link ars408_front_center" />
 
-      <include file="$(find-pkg-share pe_ars408_ros)/launch/continental_ars408.launch.xml">
-        <arg name="input/frame" value="$(var input/frame)" />
-        <arg name="output/objects" value="$(var output/objects)" />
-        <arg name="launch_driver" value="$(var launch_driver)" />
+      <include file="$(find-pkg-share common_sensor_launch)/launch/ars408.launch.xml">
+        <arg name="interface" value="can0" />
+        <arg name="input/frame" value="from_can_bus" />
+        <arg name="output_frame" value="ars408" />
       </include>
 
-    </group>
```

Comment:
I think we need 
`    </group>`
here.

Reply:
fixed at [7b05d8d](https://github.com/tier4/aip_launcher/pull/155/commits/7b05d8d24e066562b0bf641f7db383459ea463fc)

Reply:
> Do we need to keep this up?
> if="$(eval '&quot;$(var perception_mode)&quot;==&quot;camera_lidar_radar_fusion&quot;')

We do not need to keep it for now.
When can0 and can1 are defined by communication to vehicle and radar, different CAN information is entered into the driver in the vehicle without radar.
But CAN interface is fixed for now, so switching mode do not need and I fixed at [56af2aa](https://github.com/tier4/aip_launcher/pull/155/commits/56af2aa10a9d47ab1949570b53fa89edba50a929).